distributed/client.py

```diff
@@ -5514,20 +5514,26 @@ class performance_report:
     mode: str, optional
         Mode parameter to pass to :func:`bokeh.io.output.output_file`. Defaults to ``None``.
 
-    Examples
-    --------
-    >>> with performance_report(filename="myfile.html", stacklevel=1):
-    ...     x.compute()
+    **storage_options:
```

Comment:
```suggestion
    storage_options: dict, optional
```

Reply:
Yeah, no worries 👍

Reply:
I finished working on this PR, but tests seem to be failing and I believe it's not related to the code I've introduced.